ci/citest/acceptance-test/tests/cmd_template_test.go

```diff
@@ -0,0 +1,27 @@
+// +build acceptance
+
+package tests
+
+import (
+	"testing"
+)
+
+//go:generate go-bindata -pkg tests -o fixtures.bindata.go ./fixtures
+
+func init() {
+	if err := RestoreAsset("/var/tmp", "fixtures/lxc.json"); err != nil {
+		panic(err)
+	}
+}
+
+func TestCmdTemplateValidate(t *testing.T) {
+	RunCmdAndReportFail(t, "openvdc", "template", "validate", "centos/7/lxc")
+	RunCmdAndReportFail(t, "openvdc", "template", "validate", "/var/tmp/fixtures/lxc.json")
+	RunCmdAndReportFail(t, "openvdc", "template", "validate", "https://raw.githubusercontent.com/axsh/openvdc/master/templates/centos/7/lxc.json")
+}
+
+func TestCmdTemplateShow(t *testing.T) {
+	RunCmdAndReportFail(t, "openvdc", "template", "show", "centos/7/lxc")
+	RunCmdAndReportFail(t, "openvdc", "template", "show", "/var/tmp/fixtures/lxc.json")
+	RunCmdAndReportFail(t, "openvdc", "template", "show", "https://raw.githubusercontent.com/axsh/openvdc/master/templates/centos/7/lxc.json")
```

Comment:
Here I don't think it is enough to just check the exit status. What if the commands don't show anything or show something completely wrong and still exit with 0?